cognee/api/v1/datasets/routers/get_datasets_router.py

```
@@ -0,0 +1,149 @@
+        from cognee.shared.utils import render_graph
+        from cognee.infrastructure.databases.graph import get_graph_engine
+
+        try:
+            graph_client = await get_graph_engine()
+            graph_url = await render_graph(graph_client.graph)
+
+            return JSONResponse(
+                status_code=200,
+                content=str(graph_url),
+            )
+        except:
```

Ruff

Do not use bare `except`

(E722)